### ISSUE TITLE
Make clicks on label change checkbox value

### ISSUE THREAD

**Issue by agrodet** [Tatoeba/tatoeba2#2134]
Let's take the advanced search form, for example. I think that clicking on "Reverse order" should modify the checkbox value. "Owned by a self-identified native" has a similar behavior, and probably other `md-checkbox`, I haven't check.

See this piece of code
https://github.com/Tatoeba/tatoeba2/blob/f070d7d4b574f8609b8332d70204741243f77aa6/src/Template/Element/advanced_search_form.ctp#L318-L334

**Comment by trang** [comment]
It should be just a matter of having the label text directly inside the <md-checkbox> tags.

See [demo for checkboxes](https://material.angularjs.org/1.1.1/demo/checkbox) on the AngularJS Material website.

**Comment by agrodet** [comment]
Hello. I could open a PR for this, but before that, let me talk about a funny thing I discovered. If we write 
```
<md-checkbox><label>blabla</label></md-checkbox>
```
the `md-checkbox` tag takes the whole space, including the checkbox and the label (obviously). However, when hovering the space inside it that is not covered by the `label`, the pointer will change to the finger pointer (default for `md-checkbox`es) but will stay the default pointer when hovering the label. 
It's a little bit difficult to describe, and because it involves pointers, it's complicated to take screenshots. For illustration, let's look at https://tatoeba.org/eng/users/login. There is a space inside the `md-checkbox` that is not covered by the `label`. Depending on your OS + browser combination, this space differs, but it gives a slight flickering effect. 

Long story short, I suggest to modify this combination of tags by simply adding the text of the label inside the `md-checkbox` and dropping the label tag, like in the demo you linked. That would give something like 
```
<md-checkbox>blabla</md-checkbox>
```

What do you think?

**Comment by trang** [comment]
> That would give something like
> ```
> <md-checkbox>blabla</md-checkbox>
> ```
> What do you think?

Yes, this is what I meant when I said "It should be just a matter of having the label text directly inside the tags." 🙂
